sopel/tools/jobs.py

```diff
@@ -1,100 +1,79 @@
 # coding=utf-8
+"""Sopel's Job Scheduler: internal tool for job management.
+
+.. note::
+
+    As of Sopel 5.3, :mod:`sopel.tools.jobs` is an internal tool. Therefore,
+    it is not shown is the public documentation.
```

Comment:
```suggestion
    it is not shown in the public documentation.
```

Also, is the blank line below this required for Sphinx to behave?

Reply:
Fixed!